src/klr.host/Bootstrapper.cs

```diff
@@ -59,6 +59,7 @@ public Task<int> Main(string[] args)
 #else
             string applicationBaseDirectory = AppContext.BaseDirectory;
 #endif
+            Console.WriteLine("!!! AppBase: " + applicationBaseDirectory);
```

Comment:
It was a temporary backup and that line was for testing & debugging. Please check the latest commit.